qcodes/utils/plotting.py

```diff
@@ -217,27 +219,30 @@ def auto_color_scale_from_config(colorbar: matplotlib.pyplot.colorbar,
                                  ) -> None:
     """
     Sets the color limits such that outliers are disregarded, depending on
-    the configuration file `qcodesrc.json`.
-
-    Config:
```

Comment:
@Dominik-Vogel This is not a valid google style header. I have rewritten it more traditionally. Is that ok? Otherwise we can come up with something smarter that tries to match what you had but is still correct

Reply:
I am fine with this. I just did it like this to make it easier to read and considered the non conform part as part of the main doc string...